index.rst

```diff
@@ -471,6 +494,42 @@ retrieved by adding ``_`` plus the extension to the dataset type when calling
 * ``<datasetType>_visitInfo`` for example
   ``wcs = butler.get(calexp_visitInfo, ...)``
 
+Bypass Functions
+^^^^^^^^^^^^^^^^
+
+If a ``CameraMapper`` subclass wants to use a different deserializer than the
+standard Butler deserialization schemes, it can implement a function that starts
+with ``bypass`` and ends with the datasetType name. The function signature must
+be ``(self, datasetType, pythonType, location, dataId)`` where ``datasetType``
+is the datasetType name (matches the policy), ``pythonType`` is a class instance
+that the policy specifies for the datasetType's python type, ``location`` is a
+``ButlerLocation``, and ``dataId`` is the the ``dataId`` that was used to map
```

Comment:
"``dataId`` is the the ``dataId``" exceeds my recursion limit :scream_cat: Not sure what to suggest here, maybe "``dataId`` is the the ``DataId`` instance"?